docker-compose-separate-containers.yml

```diff
@@ -0,0 +1,34 @@
+version: '2'
+services:
+  nginx:
+    image: nginx
+    container_name: nginx
+    ports:
+      - "80:80"
+    volumes:
+      - /tmp/nginx:/etc/nginx/conf.d
```

Comment:
This is necessary for nginx to have access to the generated conf file. I suspect I'm missing your point.